src/transformers/tokenization_utils_base.py

```diff
@@ -126,6 +126,8 @@ class EncodingFast:
 PreTokenizedInputPair = Tuple[List[str], List[str]]
 EncodedInputPair = Tuple[List[int], List[int]]
 
+# Define type aliases for text-related non-text modalities
+AudioInput = Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput], List[float]]
```

Comment:
This is a bit confusing to me - why would an audio input be defined by text types?

Reply:
The main reason is that in some models, it indeeds  expects token sequences as input - 100% agreed it's confusing, what about a base like `audio (`np.ndarray`, `torch.Tensor`, `List[np.ndarray]`, `List[torch.Tensor]`)` ?

Reply:
This I like more! :)